modules/soe_paragraph_cta_list/config/install/paragraphs.paragraphs_type.cta_list.yml

```diff
@@ -0,0 +1,14 @@
+uuid: 720f8414-f928-430d-b01f-5691a02dcf56
+langcode: en
+status: true
+dependencies:
+  content:
+    - 'file:file:c3718e90-6b13-40c4-961a-bb0a28f6ad1c'
+_core:
+  default_config_hash: '-PxPJz2He0UqrQXYVGm9QVGMw9nrviZLOmcSPaE4_Bw'
+id: cta_list
```

Comment:
Paragraphs should follow the naming convention with prefix `stanford_`
https://docs.google.com/spreadsheets/u/1/d/1ATP3boyFio_s2BYjvzBoOt29p7aQrQV3HB-kZOZwp0g/edit#gid=0

Reply:
I've renamed all the paragraphs to follow this convention.